en/docs/get-started/streaming-quick-start-guide.md

```diff
@@ -1,142 +1,221 @@
 
-    `INFO {org.wso2.carbon.siddhi.editor.core.internal.WorkspaceDeployer} - Siddhi App ManageProductionStats.siddhi successfully deployed.`
+    ```INFO {org.wso2.carbon.streaming.integrator.core.internal.StreamProcessorService} - Siddhi App ManageProductionStats1 deployed successfully```
 
-## Step 3: Install the required extensions
 
-The `ManageProductionStats` Siddhi application uses a Kafka sink. However, the Siddhi extension for Kafka is not installed by default. To install it so that the Siddi application can integrate with Kafka as expected, follow the steps below:
+### Step 4: Test your Siddhi application
 
-1. Navigate to the `<SI_HOME>/bin` directory and issue the appropriate command based on your operating system:
+To test the `ManageProductionStats` Siddhi application above created, follow the steps below.
 
-    - **For Linux**: `./extension-installer.sh install`
-    - **For Windows**: `extension-installer.bat install`
+1. Copy the `sampledata.xml` file in the `/Users/foo/files` directory. 
 
-    As a result, a message appears in the terminal with a list of extensions used in your Siddhi application that are not completely installed, and requests you to confirm whether the system should proceed to install them
+2. Observe the SI console output.
 
+You can see the following message in the SI console log.
 
-2. Enter `Y` in the terminal to confirm that you want to proceed to install the required extensions, and then press the return key. Then the following message is displayed to indicate that the extension is installed.
+```jvm
+    INFO {io.siddhi.core.stream.output.sink.LogSink} - ManageProductionStats1 : ProductionAlertStream : [Event{timestamp=1630491334294, data=[Almond cookie, 170.0], isExpired=false}, Event{timestamp=1630491334294, data=[Baked alaska, 100.0], isExpired=false}, Event{timestamp=1630491334294, data=[Toffee, 100.0], isExpired=false}] 
+```
 
-    ```text
-    Installing kafka ...
-    Installation completed with status: INSTALLED.
+</br>
+ Congratulations!
+ Now, you have created a your first Streaming service. You may follow steps given below to expose the service as a Managed API in API Manager.
+      
+## Exposing an Streaming Service as a Managed API
 
-    Installed missing extension 1 of 1.
-    Please restart the server.
-    ```
+The `ManageProductionStats` Siddhi Application you deployed in the Micro Integrator is a **streaming service** for the API Manager. Now, let's look at how you can expose the streaming service to the API Management layer and generate a managed API by using the service.
 
-3. Restart the WSO2 Streaming Integrator server as instructed.
+### Step 1 - Expose as a service 
 
+1. Start the API Manager runtime:
 
-## Step 4: Start Kafka and create a topic
+    1.  Go to the [WSO2 API Manager website](https://wso2.com/api-management/), click **TRY IT NOW**, and then click **Zip Archive** to download the API Manager distribution as a ZIP file.
+    2.  Extract the ZIP file.
```

Comment:
These should be added to the Before you Begin section for consistency. We seem to have the download instructions there.

Reply:
because we need tis only if these spes are following.
Still need to move there as optional downloads?

Reply:
Yes, let's move it anyway. If SI is packaged inside it, it should be a mandatory download instead of SI.

Reply:
moved.